docs/ita.md

```diff
@@ -0,0 +1,201 @@
+# Intel Trust Authority (ITA)
+
+## OpenShift deployment
+
+The following instructions are assuming an OpenShift cluster is running, and Trustee Operator already installed.
+
+### Check Trustee Operator installation
+
+Now it is time to check if the Trustee operator has been installed properly, by running the command:
+
+```bash
+oc get csv -n trustee-operator-system
+```
+
+We should expect something like:
+
+```bash
+NAME                      DISPLAY            VERSION   REPLACES   PHASE
+trustee-operator.v0.1.0   Trustee Operator   0.1.0                Succeeded
+```
+
+### Use an ITA / ITTS capable KBS_IMAGE_NAME
+
+```bash
+oc -n trustee-operator-system edit trustee-operator.v0.1.0 
+```
+
+Search for the `KBS_IMAGE_NAME`, and replace the image used to `quay.io/fidencio/trustee:v0.10.1.1`.
+Note that the image used consists in `ghcr.io/confidential-containers/key-broker-service:ita-as-v0.10.1` (the official released image for v0.10.1), plus one patch that avoids a breaking change on the ITA / ITTS side (https://github.com/confidential-containers/trustee/commit/10c689540305b39a951511fecf90babc444eb8a5)
+
+## Configuration
+
+The Trustee Operator configuration requires a few steps. Some of the steps are provided as an example, but you may want to customize the examples for your real requirements.
+
+### Authorization key-pair generation
+
+First of all, we’d need to create the key pairs for Trustee authorization. The public key is used by Trustee for client authorization, the private key is used by the client to prove its identity and register keys/secrets.
+
+Create secret for client authorization:
+
+```bash
+openssl genpkey -algorithm ed25519 > privateKey
+openssl pkey -in privateKey -pubout -out publicKey
+oc create secret generic kbs-auth-public-key --from-file=publicKey -n trustee-operator-system
+```
+
+### Trustee ConfigMap object
+
+This command will create the ConfigMap object that provides Trustee all the needed configuration.
+
+Note: you'd probably need to change the Intel API key. Now it is set to `api_key": "tBfd5kKX2x9ahbodKV1..."`
```

Comment:
Adding a link to create an API key will be helpful imho. I'm assuming using one's own API key is must for ITA.

Reply:
I've put a link to ITA / ITTS at the beginning of the document,